cc_licenses/settings/base.py

```diff
@@ -206,3 +224,6 @@
 
 # django-distill settings
 DISTILL_DIR = f'{ROOT_DIR}/build/'
+
+# By default, assume the translation data's repo is checked out beside this one.
+TRANSLATION_REPOSITORY_DIRECTORY = os.path.join(ROOT_DIR, "..", "cc-licenses-data")
```

Comment:
Alternatively, I think it may be better to checkout `cc-licenses-data` remotely rather than on the file system using the `grequest` library. This way, we can ultimately have two decoupled repositories.

Reply:
Please let me know what you think. I'm not sure if this is do-able/advisable.

Reply:
What is `grequest`? I cannot find a pypi library by that name. Is it a library to access remote git repos?

I think it'll be simpler to work with a local checkout, at least for now.  That won't work if we ever have to scale this to multiple web servers, but since it mostly just generates static sites, its mission would have to change significantly for that to be even a possibility.